lib/class-wp-rest-menu-items-controller.php

```diff
@@ -542,17 +542,6 @@ protected function prepare_item_for_database( $request ) {
 			$prepared_nav_item[ $key ] = implode( ' ', array_map( 'sanitize_html_class', $value ) );
 		}
 
-		// Apply the same filters as when calling wp_insert_post().
-
-		/** This filter is documented in wp-includes/post.php */
-		$prepared_nav_item['menu-item-title'] = wp_unslash( apply_filters( 'title_save_pre', wp_slash( $prepared_nav_item['menu-item-title'] ) ) );
-
-		/** This filter is documented in wp-includes/post.php */
-		$prepared_nav_item['menu-item-attr-title'] = wp_unslash( apply_filters( 'excerpt_save_pre', wp_slash( $prepared_nav_item['menu-item-attr-title'] ) ) );
-
-		/** This filter is documented in wp-includes/post.php */
-		$prepared_nav_item['menu-item-description'] = wp_unslash( apply_filters( 'content_save_pre', wp_slash( $prepared_nav_item['menu-item-description'] ) ) );
-
```

Comment:
This is copied from [here](https://github.com/WordPress/wordpress-develop/blob/653790b83c335998cea5083cc3b07faa4e207d0b/src/wp-includes/customize/class-wp-customize-nav-menu-item-setting.php#L717-L726). 

Do you know why they are there?

Reply:
> Do you know why they are there?

I guess they are there to call `apply_filters` function before saving values to the database.
But that is not needed in `lib/class-wp-rest-menu-items-controller.php`  file because we apply those filters later (yes, I've checked that, please see `How has this been tested?` section).